osf/models/preprint_service.py

```diff
@@ -41,6 +41,7 @@ class PreprintService(DirtyFieldsMixin, GuidMixin, IdentifierMixin, ReviewableMi
 
     identifiers = GenericRelation(Identifier, related_query_name='preprintservices')
     preprint_doi_created = NonNaiveDateTimeField(default=None, null=True, blank=True)
+    download_count = models.IntegerField(default=0)
```

Comment:
Drive-by: Shouldn't we use the existing `PageCounter` model for this?

Reply:
@sloria It's a bullet point in the original ticket that this field be created, I'm unsure exactly why.

Reply:
Ah, those were the notes from grooming. We just forgot about the `PageCounter` class.